### PR TITLE
bump up graphql-sequelize and associated changes

### PR DESCRIPTION
**Pull request by jayprakash1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "prepublish": "npm run build",
+    "preinstall": "npm run build",
```
Comment (Glavin001) [inline package.json:10]: Why is this changed?

File: .npmignore [hunk 2]
```diff
@@ -25,7 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
-src
```
Comment (Glavin001) [inline .npmignore:28]: Why was this removed? I do want `src/` to be ignored and `lib/` to be packaged upon npm publish.
See https://github.com/Glavin001/graphql-sequelize-crud/blob/master/package.json#L5

**Review by Glavin001 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jayprakash1) [reply package.json:10]: My apologies, that shouldn't be changed. Please reject this pull request anyway as there is one test failing due to changes I did. I will send another pull request later. 

I was trying to make our project package.json depend on my forked repo and wasn't able to get it work because the way npm works. So, made this change and the change below. 

Your work on this project is really helpful. So thank you.

Comment (Glavin001) [reply package.json:10]: > Your work on this project is really helpful. So thank you.

Glad to hear! You're very welcome. 😃 

> I will send another pull request later.

Sounds good. Let me know if you have any questions!
